components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -129,16 +132,20 @@ func main() {
 
 	// create and run queue, steps provisioning
 	initialisation := provisioning.NewInitialisationStep(db.Operations(), db.Instances(), provisionerClient, directorClient, inputFactory, cfg.ManagementPlaneURL)
+
 	resolveCredentialsStep := provisioning.NewResolveCredentialsStep(db.Operations(), accountProvider)
+	provisionAzureEventHub := event_hub.NewProvisionAzureEventHubStep(db.Operations(), azure.NewAzureClient(), accountProvider, ctx)
 	runtimeStep := provisioning.NewCreateRuntimeStep(db.Operations(), db.Instances(), provisionerClient)
 	smOverrideStep := provisioning.NewServiceManagerOverridesStep(db.Operations(), cfg.ServiceManager)
 
 	logs := logrus.New()
 	stepManager := process.NewManager(db.Operations(), logs)
 	stepManager.InitStep(initialisation)
+
 	stepManager.AddStep(1, resolveCredentialsStep)
-	stepManager.AddStep(2, smOverrideStep)
+	stepManager.AddStep(2, provisionAzureEventHub)
```

Comment:
is it also valid for GCP provider?

Reply:
Hi PM, 
yes currently this is the default eventing system for managed offering. so it's applicable for all.

Reply:
So if I'm spinning Kyma Runtime in GCP azure event hub also will be provisioned, right?

Reply:
yup, till we come up with an alternative for GCP. or something generic for both ;)